crates/re_renderer/src/mesh.rs

```diff
@@ -198,17 +196,11 @@ impl GpuMesh {
                 },
             );
             let mut staging_buffer = queue
-                .write_buffer_with(
-                    pools
-                        .buffers
-                        .get_resource(&index_buffer)
-                        .map_err(ResourceManagerError::ResourcePoolError)?,
-                    0,
-                    index_buffer_size.try_into().unwrap(),
-                )
+                .write_buffer_with(&index_buffer, 0, index_buffer_size.try_into().unwrap())
                 .unwrap(); // Fails only if mapping is bigger than buffer size.
```

Comment:
unrelated, but there is a lot of scary `unwrap`s in this code

Reply:
all  of those `write_buffer_with` will soon be replaced with the staging belt, making them at least look less scary 🙃